### PR TITLE
Adds input capping and sanitization where needed

### PR DESCRIPTION
**Pull request by qwertyquerty**
# About The Pull Request

Adds input capping in a bunch of places

## Why It's Good For The Game

So people can't feed byond massive strings

## Changelog
:cl:
fix: Added input capping and sanitization
config: Added some new input wrappers
/:cl:

### CODE REVIEW

File: code/game/objects/structures/musician.dm [hunk 1]
```diff
@@ -292,7 +292,7 @@
 
 	else if(href_list["modifyline"])
 		var/num = round(text2num(href_list["modifyline"]),1)
-		var/content = stripped_input(usr, "Enter your line: ", instrumentObj.name, lines[num], MUSIC_MAXLINECHARS)
+		var/content = stripped_input(usr, "Enter your line: ", instrumentObj.name, lines[num])
```
Comment (ike709) [inline code/game/objects/structures/musician.dm]: :thinking:

Comment (qwertyquerty) [reply code/game/objects/structures/musician.dm]: oops

**Review by PowerfulBacon — DISMISSED** [review]
The bit I am a code owner of is fine.

File: code/__HELPERS/text.dm [hunk 2]
```diff
@@ -93,15 +93,32 @@
...
+	else
+		return trim(name, max_length)
+
+/// Used to get a properly maximum length capped input, but this time multiline.
+/proc/capped_multiline_input(mob/user, message = "", title = "", default = "", max_length=MAX_MESSAGE_LEN, no_trim=FALSE)
+	var/name = input(user, message, title, default) as text|null
+	if(no_trim)
+		return copytext(name, 1, max_length)
+	else
+		return trim(name, max_length)
```
Comment (PowerfulBacon) [inline code/__HELPERS/text.dm]: Is there any difference between these 2 procs apart from the name?

**Review by ike709 — APPROVED** [review]

**Review by zeskorion — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**